ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -206,38 +206,39 @@ private final class SqlLedger(
       val (transactionForIndex, disclosureForIndex, globalDivulgence) =
         Ledger.convertToCommittedTransaction(transactionId, transaction)
 
+      val ledgerTime = transactionMeta.ledgerEffectiveTime.toInstant
       val recordTime = timeProvider.getCurrentTime
-      val entry = if (recordTime.isAfter(submitterInfo.maxRecordTime.toInstant)) {
-        // This can happen if the DAML-LF computation (i.e. exercise of a choice) takes longer
-        // than the time window between LET and MRT allows for.
-        // See https://github.com/digital-asset/daml/issues/987
-        PersistenceEntry.Rejection(
-          LedgerEntry.Rejection(
-            recordTime,
-            submitterInfo.commandId,
-            submitterInfo.applicationId,
-            submitterInfo.submitter,
-            RejectionReason.TimedOut(
-              s"RecordTime $recordTime is after MaximumRecordTime ${submitterInfo.maxRecordTime}")
-          )
-        )
-      } else {
-        PersistenceEntry.Transaction(
-          LedgerEntry.Transaction(
-            Some(submitterInfo.commandId),
-            transactionId,
-            Some(submitterInfo.applicationId),
-            Some(submitterInfo.submitter),
-            transactionMeta.workflowId,
-            transactionMeta.ledgerEffectiveTime.toInstant,
-            recordTime,
-            transactionForIndex,
-            disclosureForIndex
+      val timeModel: TimeModel = ???
```

Comment:
Probably we should keep a `@volatile var` reference to the time model inside `SqlLedger`, whenever it is persisted to have easy acccess to it: https://github.com/digital-asset/daml/blob/f67c32b002fd28a68e6cfd279ec1782db4a07336/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala#L327-L328